src/api/instance.ts

```diff
@@ -1,22 +1,19 @@
 import ky from 'ky';
-import cookie from 'js-cookie';
+import { setAuthorizationHeader } from '@/utils/ky/hooks/beforeRequest';
+import {
+  retryRequestOnUnauthorized,
+  throwServerErrorMessage
+} from '@/utils/ky/hooks/afterResponse';
+
 const api = ky.create({
-  prefixUrl: 'http://152.67.202.74:9090/v1',
+  prefixUrl: 'http://3.34.243.139:8080/v1',
```

Comment:
env 파일로 관리하는 게 좋을 것 같아요!
```suggestion
  prefixUrl: process.env.NEXT_PUBLIC_SERVER_ENDPOINT+'/v1',
```
env.local
`NEXT_PUBLIC_SERVER_ENDPOINT='http://3.34.243.139:8080'`

env.production
`NEXT_PUBLIC_SERVER_ENDPOINT='https://프로덕션_도메인'`

Reply:
이부분 vercel 배포 환경변수에도 추가해두겠습니다~